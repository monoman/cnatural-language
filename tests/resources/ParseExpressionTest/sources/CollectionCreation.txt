new List<int> { 1 }