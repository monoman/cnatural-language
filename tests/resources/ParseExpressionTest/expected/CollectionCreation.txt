new List<int>{{Literal(DecimalInteger, [1])}}